test/index.js

```diff
@@ -732,4 +825,53 @@ lab.experiment('Normal setup', function (done) {
         });
     });
 
+    lab.test('policy can be added programmatically', function (done) {
+        server.plugins.mrhorse.addPolicy('yetAnotherPolicy', (request, reply, callback) => callback(null, true));
```

Comment:
Please add a negative assertion first to make sure the 'yetAnotherPolicy' policy doesn't exist at first.